lib/fastlane/actions/mailgun.rb

```diff
@@ -0,0 +1,136 @@
+module Fastlane
+  module Actions
+    class MailgunAction < Action
+      def self.git_author
```

Comment:
This is taken from `slack.rb`, right? I should move that somewhere so all actions can use it.